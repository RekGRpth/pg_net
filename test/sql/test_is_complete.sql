begin;

    with req as (
        select http.async_get('http://supabase.io') as id
    )
    select
        http.is_complete(id)
    from
        req;

rollback;
