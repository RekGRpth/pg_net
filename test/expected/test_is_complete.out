begin;
    with req as (
        select net.async_get('net://supabase.io') as id
    )
    select
        net.is_complete(id)
    from
        req;
 is_complete 
-------------
 f
(1 row)

rollback;
